src/cavendish_particle_tracks/_analysis.py

```diff
@@ -89,7 +89,7 @@ def _vars_to_show(self, calibrated=False):
             ]
         else:
             return [
-                "Name",
+                "name",
```

Comment:
Could we use [`dataclass.fields(ParticleDecay())`](https://docs.python.org/3/library/dataclasses.html#dataclasses.fields) here? Then we don't have to repeat ourselves?

Reply:
This function decides what fields are displayed int the table, which is a subset of all of the attributes of the class.